tuna/import_db.py

```diff
@@ -211,6 +213,11 @@ def print_sqlite_rows(cnx, cfg_filter):
 def main():
   """main"""
   args = parse_args()
+
+  args.label = "imported perf db"
+  args.arch, args.num_cu = parse_pdb_filename(args.target_file)
```

Comment:
keep in mind this is going to need utest - if this already exists, it will need to be updated

Reply:
utest added for function